props/saber_fett263_buttons.h

```diff
@@ -293,248 +664,3398 @@ class SaberFett263Buttons : public virtual PropBase {
+        }
+        break;
+      case MENU_STYLE_OPTION:
+      case MENU_IGNITION_OPTION:
+      case MENU_RETRACTION_OPTION:
+      case MENU_PREON_OPTION:
+        calc_ += direction;
+        if (calc_ > 32768) calc_ = 0;
+        if (calc_ < 0) calc_ = 32768;
+        SetInOut();      
+        break;
+      case MENU_LOCKUP_POSITION:
+      case MENU_EMITTER_SIZE:
+      case MENU_PREON_SIZE:
+        if (direction > 0) {
+          calc_ += 1000;
+          PlayMenuSound("mup.wav");
+        } else {
+          if (calc_ > 0) {
+            calc_ -= 1000;
+            PlayMenuSound("mdown.wav");
+          } else {
+            if (calc_ < 1000) {
+              calc_ = 0;
+              PlayMenuSound("mmin.wav");
+            }
+          }
+        }
+        IntEdit::SetIntValue(calc_);
+        break;
+      case MENU_DRAG_SIZE:
+        if (direction > 0) {
+          if (calc_ > 1000) {
+            calc_ -= 1000;
+            PlayMenuSound("mup.wav");
+          } else {
+            if (calc_ < 1000) {
+              calc_ = 0;
+              PlayMenuSound("mmax.wav");
+            }
+          }
+        } else {
+          if (calc_ < 32768) {
+            calc_ += 1000;
+            PlayMenuSound("mdown.wav");
+          } else {
+            if (calc_ >= 32768) {
+              calc_ = 32768;
+              PlayMenuSound("mmin.wav");
+            }
+          }
+        }
+        IntEdit::SetIntValue(calc_);
+        break;
+  #endif            
+      }
+    } 
+
+// Edit Mode Undo (AUX Button)
+  void MenuUndo() {
+    switch (menu_type_) {
+      case MENU_TOP:
+        PlayMenuSound("mexit.wav");
+        MenuExit();
+        break;
+      case MENU_PRESET:
+        first_preset();
+        PlayMenuSound("mselect.wav");
+        menu_type_ = MENU_TOP;
+        menu_ = false;
+        break;
+#ifdef FETT263_EDIT_MODE_MENU
+      case MENU_VOLUME:
+        menu_type_ = MENU_SETTING_SUB;
+        dynamic_mixer.set_volume(VOLUME);
+        MenuRevert();
+        break;
+#if NUM_BLADES > 1
+      case MENU_BLADE_STYLE:
+        menu_type_ = MENU_TOP;
+        show_preview_.Stop(blade_preview_);
+        blade_preview_ = 0;
+        MenuCancel();
+        break;
+      case MENU_BLADE_COLOR:
+        menu_type_ = MENU_TOP;
+        show_preview_.Stop(blade_preview_);
+        blade_preview_ = 0;
+        MenuCancel();
+        break;
+      case MENU_BLADE_COPY:
+        menu_type_ = MENU_COLOR_SUB;
+        show_preview_.Stop(blade_preview_);
+        blade_preview_ = 0;
+        MenuCancel();
+        break;
+      case MENU_BLADE_LENGTH:
+        menu_type_ = MENU_SETTING_SUB;
+        show_preview_.Stop(blade_preview_);
+        blade_preview_ = 0;
+        MenuCancel();
+        break;
+      case MENU_COPY_COLOR:
+        menu_type_ = MENU_COLOR_SUB;
+        show_preview_.Stop(blade_preview_);
+        blade_preview_ = 0;
+        MenuCancel();
+        break;
+#endif
+      case MENU_STYLE:
+        char style_arg[10];
```

Comment:
When you have local variables after a case, you should have brackets, like:

case MENU_STYLE: {
   char style_arg[10];
   .....
   break;
}

Reply:
added